src/main/java/org/eln2/mc/common/damagehandlers/boomboom.kt

```diff
@@ -0,0 +1,5 @@
+package org.eln2.mc.common.damagehandlers
+
+class boomboom {
+    
+}
```

Comment:
Ditto. Also, class names must start with an uppercase letter per Java conventions.

Reply:
Removed the class.